crates/biome_js_analyze/tests/specs/nursery/useQwikValidLexicalScope/invalid.jsx

```diff
@@ -0,0 +1,21 @@
+import { component$, useTask$, $ } from '@builder.io/qwik';
+
+export const HelloWorld = component$(() => {
```

Comment:
This example appears to be too large in relation to the rule's scope.

Also, we should check for cases where the arrow functions doesn't have braces 

```js
const f = () => "bar"
```